src/applications/edu-benefits/10203/actions/post-911-gib-status.js

```diff
@@ -1,11 +1,12 @@
+// eslint-disable-next-line @department-of-veterans-affairs/use-workspace-imports
```

Comment:
Good bot.

Yes, as I mentioned earlier, I had to disable the check on this and the following import.  When I modified the import following the contents of `babel.config.js`, the linter was satisfied, but then the Cypress tests failed saying it could not find the import.

Strangely, my VSCode could cmd-click to the definition, so somehow my dev environment could see everything but Cypress couldn't?  ("Works on my machine."  Classic 😎)